vscripts/client/cl_fullscreen_map.nut

```diff
@@ -7,6 +7,8 @@ function main()
 	{
 		RegisterConCommandTriggeredCallback( "+displayFullscreenMap", FullscreenMinimapButtonPressed )
 		RegisterConCommandTriggeredCallback( "-displayFullscreenMap", FullscreenMinimapButtonReleased )
+
+		RegisterConCommandTriggeredCallback( "+toggleFullscreenMap", ToggleFullscreenMinimap )
```

Comment:
```suggestion
		RegisterConCommandTriggeredCallback( "+toggleFullscreenMap", ToggleFullscreenMinimap ) // BME
```
that's the little convention I have to make it easy on a sight to see that it was added by the mod